tests/libvmi/factory.go

```diff
@@ -51,11 +53,29 @@ func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return New(NamespaceTestDefault, RandName(DefaultVmiName), opts...)
 }
 
+// NewCirros instantiates a new Cirros based VMI configuration,
+// building its extra properties based on the specified With* options.
+func NewCirros(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	// Without a cloud-init section, the VM gets stuck on the boot, waiting to
+	// receive an IP from a DHCP server. When cloud-init is used, it gets an IP
+	// immediatelly.
+	echo := `#!/bin/bash
+	echo `
+
+	cirrosOptions := append(
+		defaultOptions(),
+		WithResourceMemory("64M"),
+		WithRng(),
+		WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskCirros)),
+		WithCloudInitNoCloudUserData(echo, true),
+	)
+	opts = append(cirrosOptions, opts...)
+	return New(NamespaceTestDefault, RandName(DefaultVmiName), opts...)
+}
+
```

Comment:
This is being already done at https://github.com/kubevirt/kubevirt/pull/4035

Reply:
Thief!

Reply:
If we can make this PR in this week, we'd need to rebase Miguels PR. Or the other way around if Miguel is faster.